.travis.yml

```diff
@@ -1,62 +1,103 @@
+sudo: false
 language: android
 
-android:
-  components:
-    - tools
-    - platform-tools
-    - build-tools-27.0.0
-    - android-16
-    - android-19
-    - android-21
-    - android-27
-    - extra-android-m2repository
-    - sys-img-armeabi-v7a-android-16
-    - sys-img-armeabi-v7a-android-19
-    - sys-img-armeabi-v7a-android-21
-
-before_cache:
-  - rm -f  $HOME/.gradle/caches/modules-2/modules-2.lock
-  - rm -fr $HOME/.gradle/caches/*/plugin-resolution/
-cache:
-  directories:
-    - $HOME/.gradle/caches/
-    - $HOME/.gradle/wrapper/
-    - $HOME/.android/build-cache
+jdk:
+- oraclejdk8
 
 env:
-  matrix:
-    - ANDROID_TARGET=android-16 ANDROID_ABI=armeabi-v7a
-    - ANDROID_TARGET=android-19 ANDROID_ABI=armeabi-v7a
-    - ANDROID_TARGET=android-21 ANDROID_ABI=armeabi-v7a
   global:
-    - ADB_INSTALL_TIMEOUT=5 # increase from default of 2 mins
+  - ADB_INSTALL_TIMEOUT=8 # increase from default of 2 mins
```

Comment:
Are these sorts of time increases necessary? The following PR passes mazerunner on CI using much lower values for both the install timeout, and the `RUNNING_CI` variable: https://github.com/bugsnag/bugsnag-android/pull/371

Reply:
On the lower API level builds, it was occasionally an issue